src/cupie.c

```diff
@@ -0,0 +1,879 @@
+    {0xfe60, 0xffd8, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfdc0, 0xfff0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfdc0, 0x0010, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfe60, 0x0028, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xff60, 0x0040, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x00a0, 0x0040, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x01a0, 0x0028, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0240, 0x0010, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0240, 0x0010, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x01a0, 0x0028, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x00a0, 0x0040, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xff60, 0x0040, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfe60, 0x0028, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfdc0, 0x0010, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfdc0, 0xfff0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xfe60, 0xffd8, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xff60, 0xffc0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x00a0, 0xffc0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x01a0, 0xffd8, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0240, 0xfff0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0000, 0x0000, 0x0000, 0x0000, 0x00, 0x00, 0x00, 0x00},
+};
+
+static const struct Unk_08353510 gUnk_083532C8[] = 
+{
+    {0xffe0, 0x00e0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffc8, 0x00c0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffa0, 0x0060, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffa0, 0xffa0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffc8, 0xff40, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffe0, 0xff20, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0020, 0xff20, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0038, 0xff40, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0060, 0xffa0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0060, 0x0060, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0038, 0x00c0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0020, 0x00e0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0020, 0x00e0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0038, 0x00c0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0060, 0x0060, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0060, 0xffa0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0038, 0xff40, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0020, 0xff20, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffe0, 0xff20, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffc8, 0xff40, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffa0, 0xffa0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffa0, 0x0060, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffc8, 0x00c0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0xffe0, 0x00e0, 0x0000, 0x0000, 0x10, 0x00, 0x00, 0x00},
+    {0x0000, 0x0000, 0x0000, 0x0000, 0x00, 0x00, 0x00, 0x00},
+};
```

Comment:
The first 2 fields of `struct Unk_08353510` are `s16` so some entries here are negative.